unlock-protocol-com/blog/unlock-fiat-subscription.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Unlock adds support for fiat NFT subscriptions
+authorName: searchableguy
+publishDate: September 27, 2022
+description: Unlock adds support for fiat NFT subscriptions
+image: /images/blog/unlock-fiat-subscription/main.png
+---
+
+![Fiat subscription confirmation page](/images/blog/unlock-fiat-subscription/main.png)
+
+We recently launched support for [crypto based NFT subscriptions](./recurring-subscription-nft.md) which allowed people to pre-approve an amount in erc20 to be charged at a regular interval set in the membership contract.
+
+Today, we are extending the same support to fiat payment options we provide (including Credit card, [Apple and Google Pay](/blog/support-for-google-and-apple-pay)). Creators can now add `recurringPayments: number` to the `paywallConfig` when building their checkout URL in order to charge their users on a regular frequency.
```

Comment:
needs more detail here. someone coming to this page may not know how to add "recurringPayments: number" to the "paywallConfig" — please add more "how to" detail here.

also, "credit card" should be lower case

Reply:
Done.